src/riakc_poolboy.app.src

```diff
@@ -2,7 +2,7 @@
 {application, riakc_poolboy,
  [
   {description, "Simple Riak PB connection pool"},
-  {vsn, "1.3.1"},
+  {vsn, "1.4.0"},
```

Comment:
Wouldn't it be user-friendlier to mirror (at least the major) version of riakc?

This is an actual question - I don't really have a strong opinion :smiley: